doc/source/ray-core/compiled-graph/quickstart.rst

```diff
@@ -0,0 +1,129 @@
+Quickstart
+==========
+
+Hello World
+-----------
+Let's start from the "hello world" example of Ray Compiled Graph. First, install Ray.
+
+.. testcode::
+
+    pip install "ray[adag]"
+
+Let's start by creating a very simple actor that directly returns a given input using classic Ray Core APIs, ``remote`` and ``ray.get``.
+
+.. testcode::
+
+    import ray
+    import time
+
+    @ray.remote
+    class SimpleActor:
+    def echo(self, msg):
+        return msg
+
+    a = SimpleActor.remote()
+
+    # warmup
+    for _ in range(5):
+        msg_ref = a.echo.remote("hello")
+        ray.get(msg_ref)
+    
+    start = time.perf_counter()
+    msg_ref = a.echo.remote("hello")
+    ray.get(msg_ref)
+    end = time.perf_counter()
+    print(f"Execution takes {(end - start) * 1000 * 1000} us")
+
+.. testoutput::
+
+    Execution takes 969.0364822745323 us
+
+Let's create an equivalent program using Ray Compiled Graph. Note 4 key differences from the classic Ray Core APIs.
+
+- Create a static DAG using ``with InputNode() as inp:`` context manager
+- Use ``bind`` instead of ``remote``.
+- Use ``experimental_compile`` API for compilation.
+- Use ``execute`` to execute the DAG.
+
+.. testcode::
+
+    import ray.dag
+    with ray.dag.InputNode() as inp:
+        # Note that it uses `bind` instead of `remote`
+        dag = a.echo.bind(inp)
+    # experimental_compile is the key for optimizing the performance.
+    dag = dag.experimental_compile()
+
+    # warmup
+    for _ in range(5):
+        msg_ref = dag.execute("hello")
+        ray.get(msg_ref)
+    
+    start = time.perf_counter()
+    # `dag.execute` runs the DAG and returns a future. You can use `ray.get` API.
+    msg_ref = dag.execute("hello")
+    ray.get(msg_ref)
+    end = time.perf_counter()
+    print(f"Execution takes {(end - start) * 1000 * 1000} us")
+
+.. testoutput::
+
+    Execution takes 86.72196418046951 us
+
+The performance of the same DAG is improved by 10X! It is because the function ``echo`` is cheap and thus highly affected by
+the system overhead. Due to various bookeeping and distributed protocol, the classic Ray Core APIs have
+usually 1ms+ system overhead. Because the DAG is known ahead of time, Compiled Graph can pre-allocate all necessary
```

Comment:
```suggestion
the system overhead. Due to various bookeeping and distributed protocol, the classic Ray Core APIs usually have 1ms+ system overhead. Because the DAG is known ahead of time, Compiled Graph can pre-allocate all necessary
```